### ISSUE TITLE
AppData as install path works fine

### ISSUE THREAD

**Issue by vinyldarkscratch** [Toylerrr/Flashforge-for-Cura#8]
Hi there,

First off, thanks for putting this together!  Having just got my FFCP unit, and Ultimaker Cura, it's nice to avoid additional customization and tweaking (I say, having just set up an OctoPrint server on a Raspberry Pi for this purpose).

Anyways, I would like to mention that it appears installing the files to the user's AppData folder works just fine.  Using a macOS system as my primary computer, I gave a shot at placing the files in the corresponding Application Support (macOS' version of AppData), and it functions as expected.  I also tried it on my Windows machine, and have had no problems.  Just a little tip that can help keep the configuration files in the case of a program reinstall, or without admin permissions!

In other words, copying the contents of `resources/` into the following folders work:
- Windows: %AppData%\cura\<version>\
- macOS: /Users/<username>/Application Support/cura/<version>/
- Linux: ~/.local/share/cura/<version>

(I'm using the latest edition of Cura, which is 4.8 at the time of this writing.)

**Comment by Toylerrr** [comment]
Looking back I thought I had a reason for doing it in the cdrive but I forget why. I'll reconfigure the installer and instructions to use appdata so it's not as 'invasive' to the cura install.

Thanks for the write-up!
